octicons/templatetags/octicons.py

```diff
@@ -86,6 +86,12 @@ def size(self):
             else:
                 size['height'] = self._calculate_height(self.options['width'])
 
+        # if a custom scale is provided
+        if self.options.get('scale'):
```

Comment:
What if either `height` or `width` or both are present in addition to the scale argument? Which of these would be honoured? Can you please explain the behaviour in comments here?

Reply:
If ```height``` and/or ```width``` is passed along with ```scale``` argument then the icon's ```height``` and/or ```width``` will be set to the passed values first and then it will be scaled according to values passed in ```scale```.